<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>save_student_info</class>
 <widget class="QDialog" name="save_student_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/school_logo/C:/Users/Defend/Desktop/picture/tree.jpg);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#0000ff;&quot;&gt;学生个人身份信息录入&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>426</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>手机号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="iphone_lineEdit">
      <property name="styleSheet">
       <string notr="true">
background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>491</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>吃饭喜好</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="red_radioButton">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>红油</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="white_radioButton">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>清汤</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>166</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>家庭地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>101</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>QQ 号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="qq_number_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>296</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>电子邮箱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_address_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>231</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>家庭邮编</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="postcode_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>361</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
      <property name="text">
       <string>籍    贯</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="home_from_lineEdit">
      <property name="styleSheet">
       <string notr="true">background-image: url();</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
